Auth: Fix orgrole picker disabled if isSynced user (#64033)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

| `logsContextDatasourceUi`          | Allow datasource to provide custom UI for context view                                                                                                                       |
| `lokiQuerySplitting`               | Split large interval queries into subqueries with smaller time intervals                                                                                                     |
| `individualCookiePreferences`      | Support overriding cookie preferences per user                                                                                                                               |

> | `onlyExternalOrgRoleSync`          | Enables organization roles synced with external auth providers to not be changed at all                                                                                      |

Reviewer: ```suggestion
| `onlyExternalOrgRoleSync`          |  Prohibits a user from changing organization roles synced with external auth providers                                                                                      |
```

Author: this file is generated automatically, so i will change it in the registry of the featuretoggle
